content/en/docs/configuration/configuration.md

```diff
@@ -268,11 +268,11 @@ Furthermore, Rebar3 stores some of its configuration data in `~/.config/rebar3`
 
 ## EDoc
 
-All options supported by [EDoc](http://www.erlang.org/doc/man/edoc.html#run-3) can be put in `{edoc_opts, [...]}`.
+All options supported by [EDoc](https://www.erlang.org/doc/man/edoc.html#run-3) can be put in `{edoc_opts, [...]}`.
 
 ## Escript
 
-Full details at the [escriptize command](http://www.rebar3.org/docs/commands#escriptize). Example configuration values below.
+Full details at the [escriptize command](https://www.rebar3.org/docs/commands#escriptize). Example configuration values below.
```

Comment:
we should actually use relative paths here I think since it's a local URL.

Reply:
Yup. I'll move all of them, while I'm at it.

Reply:
Hm... actually some, while not broken, are pointing to a "parent" page, as per redirection rules, it seems. Will fix some more.

Reply:
All this is now done.